RELEASE_NOTES.md

```diff
@@ -1,13 +1,16 @@
-# Cosmos SDK v0.44.3 Release Notes
+# Cosmos SDK v0.44.4 Release Notes
 
-Recently, the Cosmos-SDK team became aware of a high-severity security vulnerability that impacts Cosmos-SDK v0.43.x and v0.44.x and can result in a consensus halt. User funds are NOT at risk; however, the vulnerability can result in a chain halt. This vulnerability does not impact the current Cosmos Hub, though other Cosmos-SDK based blockchains using v0.43.x or v0.44.x may be affected and are advised to update to v0.44.2 immediately.
+This release introduces bug fixes and improvements on the Cosmos SDK v0.44 series.
```

Comment:
Change "introduces" to "includes"? The phrase "introduces bug" triggered me. Nevermind if these aren't your words.

Reply:
Not my words. :smile: